### PR TITLE
break: LANZ-2766 Move to Route53 domain to baseline, rename output

### PR DESCRIPTION
**Pull request by jstehlik-ohpen**

### CODE REVIEW

File: metadata/outputs.tf [hunk 1]
```diff
@@ -1,6 +1,5 @@
 locals {
   output_metadata = merge(local.main_ssm,
-    local.secondary_ssm,
```
Comment (jstehlik-ohpen) [inline metadata/outputs.tf:3]: this deletion removes domain baseline resources from metadata

File: example/account_metadata.tf [hunk 2]
```diff
@@ -0,0 +1,26 @@
+module "account_metadata" {
```
Comment (jstehlik-ohpen) [inline example/account_metadata.tf:1]: example module call that can be copy pasted to your module

File: README.md [hunk 3]
```diff
@@ -1,56 +1,166 @@
...
+
+## How to use
+1. Read the documentation to check which inputs and outputs are available
+2. Call module in your Terraform code
+3. Use outputs from the module in your locals and resources
+   1. If your IDE allows it, code completion can be used for module outputs, simplifying writing the code
+   2. Outputs have a default value, mostly null - in case output module is disabled, use [coalesce](https://www.terraform.io/language/functions/coalesce) to set a manual value, e.g. coalesce(module.account_metadata.account_id, var.account_id). In the example, var.account_id, only needs a value set, if module output is null.
+### Examples
+#### Output only account metadata
+Metadata is retrieved without the necessity of providing any inputs.
```
Comment (cangulomascarell) [inline README.md:58]: ❤❤

File: example/account_metadata.tf (hunk 2 above)
Comment (cangulomascarell) [reply example/account_metadata.tf:1]: @jstehlik-ohpen  I would create a section in the README.md with something like template and link to the files in the example/* folder so when developer are reading the readme can go there directly

**Review by cangulomascarell — DISMISSED** [review]

File: example/account_metadata.tf (hunk 2 above)
Comment (jstehlik-ohpen) [reply example/account_metadata.tf:1]: Great idea!
I will add it before merging.

Comment (jstehlik-ohpen) [reply example/account_metadata.tf:1]: @cangulomascarell Added link in How-to section

**Review by cangulomascarell — APPROVED** [review]

**Pull request merged**